code/style.css

```diff
@@ -1,4 +1,215 @@
 html {
-  background: #0025ff;
-  color: #fff;
+  background: #ffffff;
+  color: #242526;
 }
+/* only used in the header */
```

Comment:
An idea is to start the css with a .body{} to include all the general code you had, such as letter font and size when you use one type in the whole page. This way you don't need to repeat them in the specific parts. 
I found a webpage that helped me a lot with css organization guide line for this project. I'll copy the link here , so you can maybe check it out if (https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Organizing).